### DISCUSSION TITLE
DEV : test environnement

### DISCUSSION THREAD

**Discussion by pubpub-zz** [py-pdf/pypdf#973] [Q&A] [answered]
@MartinThoma,
To prevent too many cosmetic commits can you please re-clarify what are the command line parameters and config file used during generation.
Also, do you have some reference/recommendation about codevod to propose the best coverage : I was expecting more pdf input files to cover coverage, but not sure it's working

**Comment by MartinThoma** [comment, marked as answer]
I missunderstood your "convert to discussion" comment :sweat_smile: I've now answered here: https://github.com/py-pdf/PyPDF2/discussions/974 

Would it be ok for you if I delete the discussion here?
